### PR TITLE
Add example of a categorical map plot

### PR DESCRIPTION
**Pull request by AnnikaLau**

### CODE REVIEW

File: categorical_map/categorical_map.ipynb [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+   "metadata": {},
+   "source": [
+    "This plot shows pollen health impact classes over the Alps on February 15, 2020 at 8 o'clock in the evening. It shows hourly values for Alder (Alnus) pollen and has been kindly provided by Simon Adamov. The field is generated by the COSMO-1E weather model, on the typical COSMO-1E domain over the greater alpine area. For better understanding, the numerical pollen concentrations are converted into health impact classes. These classes are then presented to the public via animated pollen maps in the MeteoSwiss App."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
```
Comment (victoria-cherkas) [inline categorical_map/categorical_map.ipynb:14]: Could we add the output to the repo? I think it is nice if people are browsing the repository and they dont need to run the notebook just see clearly what it is doing from the output. What do you think? On the downside it makes git diff much more chaotic when we include output from notebooks to git.

File: categorical_map/categorical_map.ipynb [hunk 2]
```diff
@@ -0,0 +1,125 @@
+{
```
Comment (victoria-cherkas) [inline categorical_map/categorical_map.ipynb:1]: Should we add a markdown cell with the title?

**Review by victoria-cherkas — CHANGES_REQUESTED** [review]
Hey Annika looks all great to me, just added a couple of comments for readability of the repo online, if that's ok with you

File: categorical_map/categorical_map.ipynb (hunk 1 above)
Comment (AnnikaLau) [reply categorical_map/categorical_map.ipynb:16]: Thanks I added it. I also think it has advantages and disadvantages but let's be consistent with the other notebooks.

File: categorical_map/categorical_map.ipynb (hunk 2 above)
Comment (AnnikaLau) [reply categorical_map/categorical_map.ipynb:1]: done

**Comment by AnnikaLau** [comment]
@victoria-cherkas : Thanks for you review. Are you happy with my updates or anything more to change?

**Review by victoria-cherkas — APPROVED** [review]

**Comment by victoria-cherkas** [comment]
Thanks for the changes, looks good to me!

**Pull request merged**